src/lib/containers/GenericCard.tsx

```diff
@@ -438,72 +442,83 @@ export default class GenericCard extends React.Component<
+                style={{ margin: 'none' }}
+              >
+                {!titleLinkConfig &&
+                titleColumnType === ColumnType.FILEHANDLEID ? (
+                  <FileHandleLink
+                    token={token}
+                    fileHandleId={linkValue}
+                    tableEntityConcreteType={tableEntityConcreteType}
+                    showDownloadIcon={type !== SynapseConstants.EXPERIMENTAL}
+                    rowId={data![schema.id]}
+                    tableId={tableId}
+                    displayValue={title}
+                  />
+                ) : (
+                  this.renderTitle({
+                    href,
+                    target,
+                    titleSearchHandle,
+                    title,
+                  })
+                )}
+              </h3>
             </div>
-          )}
-          {/* 
-            Below is a hack that allows word highlighting to work, the Search component insert's
-            html elements outside of the React DOM which if detected would break the app,
-            but as written below this avoids that reconcilliation process.
-          */}
-          {description &&
-            this.renderShortDescription(
-              description,
-              hasClickedShowMore,
-              descriptionSubTitle,
-              descriptionLinkConfig,
-            )}
-          {description &&
-            this.renderLongDescription(
-              description,
-              hasClickedShowMore,
-              descriptionSubTitle,
-              descriptionLinkConfig,
-              this.props.token,
+            {subTitle && (
+              <div
+                data-search-handle={stubTitleSearchHandle}
+                className="SRC-author"
+              >
+                {subTitle}
+              </div>
             )}
+            {/* 
+              Below is a hack that allows word highlighting to work, the Search component insert's
+              html elements outside of the React DOM which if detected would break the app,
+              but as written below this avoids that reconcilliation process.
+            */}
+            {description &&
+              this.renderShortDescription(
+                description,
+                hasClickedShowMore,
+                descriptionSubTitle,
+                descriptionLinkConfig,
+              )}
+            {description &&
+              this.renderLongDescription(
+                description,
+                hasClickedShowMore,
+                descriptionSubTitle,
+                descriptionLinkConfig,
+                this.props.token,
+              )}
+          </div>
         </div>
         {showFooter && (
           <CardFooter
             isHeader={false}
             secondaryLabelLimit={secondaryLabelLimit}
             values={values}
             columnIconOptions={columnIconOptions}
+            className={`${imageFileHandleIdValue ? 'hasImage' : 'hasIcon'}`}
```

Comment:
Apply the class "hasImage" or "hasIcon" to footer div to change it's indent behavior (if icon, do what it does today, if image then do not try to align with the icon indent).